mmdet/utils/misc.py

```diff
@@ -74,3 +76,14 @@ def update(cfg, src_str, dst_str):
 
     update(cfg.data, cfg.data_root, dst_root)
     cfg.data_root = dst_root
+
+
+_torch_version_meshgrid_indexing = version.parse(
```

Comment:
Following the version comparison style in MMCV https://github.com/open-mmlab/mmcv/blob/235c0253ab8806a2a2ee6954b4258a95358497ac/mmcv/parallel/distributed.py#L40, it might be better to write the code as below:
```python
from mmcv.utils import TORCH_VERSION, digit_version
if ('parrots' not in TORCH_VERSION
      and digit_version(TORCH_VERSION) >= digit_version('1.10')

```

Reply:
> I can't seem to find much information on the parrots version of PyTorch - what should it do in that case? Does the `torch.meshgrid` method for parrots PyTorch support the indexing parameter or not and/or does it raise a warning?
> 
> I had a look and couldn't immediately identify what exactly `digit_version` does differently. In what case would the result of a version comparison like I coded it and your proposal differ?
> 
> `digit_version('1.10')` is not quite sufficient by the way, it does really need to be `digit_version('1.10.0a0')`. My own PyTorch installation is exactly inbetween those two and it makes a difference.

It is an internal version that we support. You are right that digit_version does not do anything different, it is just a convention. Then how about using `digit_version('1.10.0a0')`?

Reply:
I have updated the PR